trigger/build.go

```diff
@@ -294,16 +296,16 @@ func (t *buildTriggers) makeData(sub *event.Subscription, pastTenseOverride stri
 }
 
 func (t *buildTriggers) buildAttachments(data *commonTemplateData) []message.SlackAttachment {
+	hasPatch := t.build.Requester != evergreen.RepotrackerVersionRequester
```

Comment:
Should this/does this include ad hoc versions?

Reply:
Repotracker versions, ad hoc versions, and git tag versions wouldn't have a patch; I didn't include them because I don't think they have a reason to hit triggers, but it might just be worth it to cover the case now.